lib/util/jwt_utils.dart

```diff
@@ -0,0 +1,59 @@
+import 'dart:convert';
+
+/// Utility class to parse and perform JWT Operations.
+class JWTUtils {
```

Comment:
Is it possible to change the name to something like ```IoUtils``` or ```any``` that suits

Reply:
Removed the JWT Files.